Dockerfile.dapper

```diff
@@ -1,25 +1,20 @@
-FROM golang:1.16.9-buster
+FROM registry.suse.com/bci/golang:1.17
 
-ENV ARCH amd64
+ARG DAPPER_HOST_ARCH="amd64"
+ENV ARCH=${DAPPER_HOST_ARCH}
 
 ARG http_proxy=$http_proxy
 ARG https_proxy=$https_proxy
 ARG no_proxy=$no_proxy
 
-RUN apt-get update && \
-    apt-get install -y apt-transport-https ca-certificates
+RUN zypper -n in go1.17 curl bash git jq
+RUN zypper install -y -f docker
```

Comment:
This could be merged with the line above and would be good to also add `RUN zypper -n clean -a` to clean Zypper's cache.